cmd/config-v16.go

```diff
@@ -208,6 +208,11 @@ func validateConfig() error {
 		return err
 	}
 
+	if srvCfg.Version > v16 {
```

Comment:
> Actually, this is a string comparison, not integer.

`It is usually clearer and always faster to use the built-in string comparison operators ==, <, >, and so on.`

https://golang.org/pkg/strings/#Compare

> By the way, since we are always migrating older versions, we will never have the case srvCfg.Version < v16. I think this could be merge with the next if condition.

I will retain the `!=` itself then.